src/runtime/virtcontainers/clh.go

```diff
@@ -1457,6 +1519,259 @@ func (clh *cloudHypervisor) launchClh() error {
 	return nil
 }
 
+// waitForEventMonitorFile waits for the event monitor file to be created by CLH
+func (clh *cloudHypervisor) waitForEventMonitorFile(eventMonitorPath string, timeout time.Duration) error {
+	timeoutChan := time.After(timeout)
+	ticker := time.NewTicker(10 * time.Millisecond)
```

Comment:
I wouldn't check more often than every 100ms?
```suggestion
	ticker := time.NewTicker(100 * time.Millisecond)
```